src/Wizard/WizardSteps/WizardStep/WizardStep.types.ts

```diff
@@ -5,6 +5,10 @@ export type WizardStepProps = {
    * If true, the step is marked as completed. Is normally set by the WizardSteps parent component.
    */
   completed?: boolean;
+  /**
+   * If true, the step is marked as errored.
+   */
+  error?: boolean;
```

Comment:
Missing default value for this prop in storybook docs. Whilst you are there could you add defaults for the others that are missing too? I assume default is false here for new error prop.

<img width="1090" alt="image" src="https://github.com/IPG-Automotive-UK/react-ui/assets/8976377/79b19d17-7f5e-458d-a3f2-1af6f393f8d4">